docs/machine-learning/how-to-guides/matchup-app-infer-net.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Create a game match up list app with Infer.NET and probalistic programming
+description: Discover how to use probalistic programming with Infer.NET to create a game matchup list app based on a simplified version of TrueSkill.
+ms.date: 10/04/2018
+ms.topic: how-to
```

Comment:
how-to is not a valid one according to https://review.docs.microsoft.com/en-us/help/contribute/metadata-attributes?branch=master